### DISCUSSION TITLE
PSA: --xla_gpu_simplify_all_fp_conversions is about to be on by default

### DISCUSSION THREAD

**Discussion by cheshire** [openxla/xla#9] [General]
The change allows simplifying away lowering conversions (e.g. `f32->bf16->f32`), and was seen to lead to large increases in performance on some models (~10%).

This can cause casts being ignored, and overall precision higher than expected, but it's already the case for XLA compiler (e.g. elementwise ops and matmuls can be already performed in higher precision than specified in HLO)

**Comment by cheshire** [comment]
This was committed.

**Reply by nouiz** [reply]
URL: https://github.com/tensorflow/tensorflow/commit/6a98a2c6f8e4074739b8ad888a5fbd32115e105e
